automaxlair/MaxLairInstance.py

```diff
@@ -294,23 +350,44 @@ def read_selectable_pokemon(self, stage: str) -> List[Pokemon]:
+            move_2 = self.read_text(image, self.move_rect_2, threshold=False, segmentation_mode='--psm 7').strip()
+            move_3 = self.read_text(image, self.move_rect_3, threshold=False, segmentation_mode='--psm 7').strip()
+            move_4 = self.read_text(image, self.move_rect_4, threshold=False, segmentation_mode='--psm 7').strip()
+            moves.append(move_1 + move_2 + move_3 + move_4)
+            move_5 = self.read_text(image, self.move_rect_5, threshold=False, segmentation_mode='--psm 7').strip()
+            move_6 = self.read_text(image, self.move_rect_6, threshold=False, segmentation_mode='--psm 7').strip()
+            move_7 = self.read_text(image, self.move_rect_7, threshold=False, segmentation_mode='--psm 7').strip()
+            move_8 = self.read_text(image, self.move_rect_8, threshold=False, segmentation_mode='--psm 7').strip()
+            moves.append(move_5 + move_6 + move_7 + move_8)
+            move_9 = self.read_text(image, self.move_rect_9, threshold=False, segmentation_mode='--psm 7').strip()
+            move_10 = self.read_text(image, self.move_rect_10, threshold=False, segmentation_mode='--psm 7').strip()
+            move_11 = self.read_text(image, self.move_rect_11, threshold=False, segmentation_mode='--psm 7').strip()
+            move_12 = self.read_text(image, self.move_rect_12, threshold=False, segmentation_mode='--psm 7').strip()
+            moves.append(move_9 + move_10 + move_11 + move_12)
         elif stage == 'catch':
             pokemon_names.append(self.read_text(image, self.sel_rect_4, threshold=False, segmentation_mode='--psm 3').strip().split('\n')[-1])
             abilities.append(self.read_text(image, self.abil_rect_4, threshold=False, segmentation_mode='--psm 3').strip())
             types.append('')
+            move_1 = self.read_text(image, self.move_rect_13, threshold=False, segmentation_mode='--psm 7').strip()
+            move_2 = self.read_text(image, self.move_rect_14, threshold=False, segmentation_mode='--psm 7').strip()
+            move_3 = self.read_text(image, self.move_rect_15, threshold=False, segmentation_mode='--psm 7').strip()
+            move_4 = self.read_text(image, self.move_rect_16, threshold=False, segmentation_mode='--psm 7').strip()
+            moves.append(move_1 + move_2 + move_3 + move_4)
         elif stage == 'battle':
             pokemon_names.append(self.read_text(image, self.sel_rect_5, threshold=False, invert=False, segmentation_mode='--psm 8').strip())
             abilities.append('')
             type_1 = self.read_text(image, self.type_rect_1, threshold=False, invert=True, segmentation_mode='--psm 8').strip().title()
             type_2 = self.read_text(image, self.type_rect_2, threshold=False, invert=True, segmentation_mode='--psm 8').strip().title()
             types.append(type_1 + type_2)
+            moves = ['','','']

```

Comment:
Should there be 4 moves here?

Reply:
In fact, it should be that. We're not giving the moves one by one but by concatenating them (just like types)

Reply:
Ah I understand now. Looks good!